src/Animals/Cow.ts

```diff
@@ -6,7 +6,7 @@ export class Cow extends Animal {
   constructor() {
     super(
       AnimalNames.COW,
-      './static/images/avatars/cow.png',
+      './static/images/avatars/cow.svg',
```

Comment:
All these paths are coming from the configuration, not the animals itself.

Reply:
Yes, but until we will use configuration in every element, it will be better to keep this for now.

Reply:
👍